daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -193,6 +193,61 @@ APP ID     APP PORT  AGE  CREATED
 nodeapp    3000      16h  2020-07-29 17:16.22
 ```
 
+#### Upgrade from 0.11.x to 1.0.0
+
+Upgrading from 0.11.x to 1.0.0 safely requires additional step to keep 0.11.x placement service to migrate from 0.11.x placement service to 1.0.0 placement service. These two commands disable `helm upgrade` to uninstall 0.11.x placement service.
+
+```sh
+kubectl annotate deployment dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+kubectl annotate svc dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+```
+
+Then [export certs manually](#exporting-certs-manually).
+
+```sh
+dapr mtls export -o ./certs
+```
+
+Upgrade Dapr using the below commands:
+
+```sh
+helm repo update
+helm upgrade dapr dapr/dapr --version 1.0.0-rc1 --namespace dapr-system --reset-values --set-file dapr_sentry.tls.root.certPEM=./certs/ca.crt --set-file dapr_sentry.tls.issuer.certPEM=./certs/issuer.crt --set-file dapr_sentry.tls.issuer.keyPEM=./certs/issuer.key --set global.ha.enabled=true --wait
```

Comment:
This explicitly says `--version 1.0.0-rc1` will we need to update these instructions on the next RCs? perhaps it's better to change the section header to "Upgrade ... to 1.0.0-rc1"

Reply:
Or you can say, pick the v1.0 release version e.g. --version 1.0.0-rc1

Reply:
Please review it again :)